<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>doc</class>
 <widget class="QWidget" name="doc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>doc</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Add Item:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="addItemName"/>
     </item>
     <item>
      <widget class="QLineEdit" name="addItemQ"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="addButton">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Consume Item:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="consumeItemName"/>
     </item>
     <item>
      <widget class="QLineEdit" name="consumeItemQ"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="consumeButton">
     <property name="text">
      <string>Consume</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="displayButton">
     <property name="text">
      <string>Display History</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>417</width>
        <height>186</height>
       </rect>
      </property>
      <widget class="QTextEdit" name="textEdit">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-7</y>
         <width>419</width>
         <height>201</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
